### DISCUSSION TITLE
Is there a way to trigger the onUpdate event on the editor manually?

### DISCUSSION THREAD

**Discussion by dkrusenstrahle** [ueberdosis/tiptap#1275] [General Questions]
Hello,

In my Tiptap 2.00-beta-44 editor I got custom node views with forms and when the user fills out the forms I can update the node.attributes but the editor onUpdate event is not triggered. Is there a way to manually trigger this event? I need it for saving to the API.

**Comment by philippkuehn** [comment]
The update event will be triggered if you update attributes of the node view with `updateAttributes`.

**Reply by dkrusenstrahle** [reply]
Yes, thank you, it worked!
